pydis_site/apps/redirect/redirects.yaml

```diff
@@ -87,10 +87,6 @@ resources_index_redirect:
   original_path: pages/resources/
   redirect_route: "resources:index"
 
-resources_resources_redirect:
-  original_path: pages/resources/<str:category>/
-  redirect_route: "resources:resources"
-
```

Comment:
There will be other missing pages now that we've consolidated the resources into one page, though I'm not sure how to create redirects for these.